### ISSUE TITLE
ion-slides in supertabs

### ISSUE THREAD

**Issue by Chris1234567899** [zyra/ionic2-super-tabs#57]
Thanks for the plugin it really does a nice job.

One thing I am currently struggling with is putting the official ion-slides component in a super-tab.

What I would like to see is a "seamless" sliding on the last (or first) ion-slide to the next tab. 
Something like:
`tab1<-->tab2.slide1<-->tab2.slide2<-->tab3`

It works more or less right now but is very glitchy. I played around with putting the ion-slides in a container, setting margins etc, but nothing seemed very promising. Is there a trick to do so already? Would be a really nice to have!

**Comment by ihadeed** [comment]
The example project has `ion-slides` in one of the pages. Not full screen, just a small thing.

**Comment by Chris1234567899** [comment]
Thanks for the quick response. Yes a non full screen slider works just fine with the sliding tabs. However if they are fullscreen, it looks like the container of the ion-slides covers the sliding event of the supertabs. If someone tries to slide to the next tab several times and directly at the edge of the screen, sometimes it works, but mostly only the ion-slides react. Hard to explain ;)

The code is just basic stuff - no changes in css or ts;
tabs.html
```
<super-tabs>
  <super-tab [root]="page1" title="First page"></super-tab>
  <super-tab [root]="page2" title="Second page"></super-tab>
  <super-tab [root]="page3" title="Third page"></super-tab>
</super-tabs>
```

page1.html
```
<ion-content padding>
  Page 1
</ion-content>
```

page2.html
```
<ion-content padding>
  <ion-slides>
    <ion-slide>Slide 1</ion-slide>
    <ion-slide>Slide 2</ion-slide>
    <ion-slide>Slide 3</ion-slide>
    <ion-slide>Slide 4</ion-slide>
  </ion-slides>
</ion-content>
```

page3.html
```
<ion-content padding>
Page 3
</ion-content>
```


I tried to disable sliding forward of the ion-slides when the ion-slides reaches the last slide, but that seems to disable as well the supertab sliding.

**Comment by Saqib92** [comment]
Issue Still exists on ionic 5.
Using `ion-slides` changes the Tabs. When I slide on ion slide it change the whole tab. 

> Ionic:
> 
>    Ionic CLI : 6.12.4
> 
> Utility:
> 
>    cordova-res (update available: 0.15.3) : 0.15.1
>    native-run                             : 1.3.0
> 
> System:
> 
>    NodeJS : v12.18.3
>    npm    : 6.14.6
>    OS     : Windows 10
